stdlib/public/runtime/ImageInspectionWin32.cpp

```diff
@@ -200,7 +200,7 @@ void swift::initializeProtocolConformanceLookup() {
   // rdar://problem/19045112
   const InspectArgs ProtocolConformancesArgs = {
     addImageProtocolConformanceBlockCallback,
-    ProtocolConformancesSection,
+    ProtocolConformancesSection
```

Comment:
I think that keeping that is better (I like the trailing commas, and it keeps the change focused).